pkgs/development/tools/build-managers/cmake/check-improper-prefix-hook.sh

```diff
@@ -12,7 +12,7 @@ cmakePcfileCheckPhase() {
             exit 1
             } 1>&2
         fi
-    done < <(find "${!outputDev}" -iname "*.pc" -print0)
+    done < <(find "${!outputDev}" -regextype posix-extended -iregex '.*\.(pc|cmake)' -print0)
 }
 
-postFixupHooks+=(cmakePcfileCheckPhase)
+postFixupHooks+=(cmakeImproperPrefixUsageCheckPhase)
```

Comment:
It finds only prefix misuses?

Reply:
it finds when prefix is prepended to an absolute path, not sure how to call this hook